### PR TITLE
upgrade ember-cli to 2.12.3

### PR DESCRIPTION
**Pull request by helloWorldCindy**
**This project uses [semver](http://semver.org), please check the scope of this pr:**

- [ ] #none# - documentation fixes and/or test additions
- [x] #patch# - backwards-compatible bug fix
- [ ] #minor# - adding functionality in a backwards-compatible manner
- [ ] #major# - incompatible API change

# CHANGELOG

Upgrade ember-cli to 2.12.3

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
-{
-  "extends": ["frost-standard"],
-  "rules": {
+module.exports = {
+  extends: 'frost-standard',
+  rules: {
```
Comment (notmessenger) [inline .eslintrc.js]: These should be removed

File: .npmignore [hunk 2]
```diff
@@ -9,6 +10,7 @@
 .ember-cli
 .eslintignore
 .eslintrc
```
Comment (notmessenger) [inline .npmignore]: Remove

File: .npmignore [hunk 3]
```diff
@@ -9,6 +10,7 @@
...
+.eslintrc.js
 .github
 .gitignore
 .jshintrc
```
Comment (notmessenger) [inline .npmignore]: Remove

File: package.json [hunk 4]
```diff
@@ -66,4 +65,4 @@
     ],
     "configPath": "tests/dummy/config"
   }
-}
\ No newline at end of file
+}
```
Comment (notmessenger) [inline package.json]: You didn't update `ember-cli-babel` or `ember-cli-htmlbars in `dependencies` section

Comment (notmessenger) [inline package.json]: Remove `ember-cli-test-loader`

File: tests/dummy/config/environment.js [hunk 5]
```diff
@@ -1,9 +1,13 @@
+/* eslint-env node */
 module.exports = function (environment) {
   var ENV = {
     APP: {},
-    rootURL: '/',
```
Comment (notmessenger) [inline tests/dummy/config/environment.js]: This should stay.   Similar entry in test configuration should be removed though

Comment (helloWorldCindy) [inline tests/dummy/config/environment.js]: Fixed.

File: package.json (hunk 4 above)
Comment (helloWorldCindy) [inline package.json]: Fixed.

Comment (helloWorldCindy) [inline package.json]: Fixed.

File: .npmignore (hunk 3 above)
Comment (helloWorldCindy) [inline .npmignore]: Fixed.

File: .npmignore (hunk 2 above)
Comment (helloWorldCindy) [inline .npmignore]: Fixed.

File: .eslintrc.js (hunk 1 above)
Comment (helloWorldCindy) [inline .eslintrc.js]: Fixed.

**Comment by notmessenger** [comment]
👍

**Pull request merged**